### PR TITLE
augmenting job triggers page

### LINKED ISSUE #2488: [2. Job commands] Add content on non cron jobs in existing job triggers page
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

Problem statement: no clarify on when to use cron vs when to use this job scheduling. Discussion w @ernestoongaro on clarifying what a non cron trigger do in dbt cloud. it would require adjustments to this page: https://docs.getdbt.com/docs/deploy/job-triggers

### What part(s) of the page would you like to see updated?

https://docs.getdbt.com/docs/deploy/job-triggers

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mirnawong1**
adding explanation on how run on schedule works

[docs project](https://www.notion.so/dbtlabs/Enhance-job-commands-and-job-trigger-content-228d3e4c5a1f485592e1bd30e16f85ad?pvs=4)

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by ernestoongaro** [comment]
Just adding my notes/todos:

- When to use `Schedule Days` vs `Cron`
- Schedule
  - enter a value between 1 and 23 (so what is midnight!??) - why would you use this?
  - at exact intervals - how to select multiple times, or half hour intervals?
- perhaps a table with the three ways would help with different examples?

File: website/docs/docs/deploy/job-triggers.md [hunk 1]
```diff
@@ -27,3 +61,15 @@ A custom cron schedule can be specified in the Job Settings page when you edit a
 6. Enter the custom cron syntax for the schedule you want.
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/job-schedule.png" title="Schedule your dbt job"/>
+
+THE BELOW IS JUST COPIED OVER TEXT - WILL AMEND SOON!
```
Comment (ernestoongaro) [inline website/docs/docs/deploy/job-triggers.md]: @mirnawong1 this will need some help, I added back in Jobs but not sure what else you had in mind :)

File: website/docs/docs/deploy/job-triggers.md [hunk 2]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+- **Continuous Integration (CI)** tab &mdash; Configure [continuous integration](/docs/deploy/cloud-ci-job) to run when someone opens a new pull request in your dbt repository
+- **API** tab &mdash; Use the [API](/docs/dbt-cloud-apis/overview) to trigger a job or send events to other systems
+
+<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/triggers.jpg" title="Configuring your job triggers"/>
+
+## Scheduling
+
+To configure your job to run on particular day(s) and time(s), you can toggle the **Run on schedule** toggle and customize the days, time, and intervals you want your job to run.
+
+### Schedule Days
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
### Schedule days
```

File: website/docs/docs/deploy/job-triggers.md [hunk 3]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+
+dbt Cloud uses [coordinated universal time](https://en.wikipedia.org/wiki/Coordinated_Universal_Time), it is not translated to your timezone or aware of daylight savings time. For examples:
+
+- 0 means 12am (midnight) UTC
+- 12 means 12pm (afternoon) UTC
+- 23 means 11pm UTC
+
+:::
+
+### Custom Cron Schedule
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
### Custom cron schedule
```

File: website/docs/docs/deploy/job-triggers.md [hunk 4]
```diff
@@ -27,3 +61,15 @@ A custom cron schedule can be specified in the Job Settings page when you edit a
...
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/job-schedule.png" title="Schedule your dbt job"/>
+
+THE BELOW IS JUST COPIED OVER TEXT - WILL AMEND SOON!
+
+6. Select **Save**, then click **Run Now** to run your job. Click the run and watch its progress under "Run history." 
+
+## Related docs
+
+- [Job Commands](/docs/deploy/job-commands)
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
- [Job commands](/docs/deploy/job-commands)
```

File: website/docs/docs/deploy/job-triggers.md [hunk 5]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
 description: "You can use cron syntax to specify when you want to run a job."
 ---
 
-In dbt Cloud, you can use "cron" syntax to specify when you'd like your job to run. Cron syntax is very expressive, and allows you to completely customize your run schedule.
+In dbt Cloud, you can configure when and how dbt should trigger your job run.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
In dbt Cloud, you can use the options under **Triggers** to configure when and how dbt should run your job:
```

File: website/docs/docs/deploy/job-triggers.md [hunk 6]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+In dbt Cloud, you can use the options under **Triggers** to configure when and how dbt should run your job:
 
-If you need help coming up with the right cron syntax, we recommend using a tool like `crontab.guru`. There, you can enter cron snippets and see what they mean in plain English. You can also find some example snippets below.
+- **Schedule** tab &mdash; Use the **Run on schedule** toggle to configure your job to run on either simple [scheduled days](#schedule-days) or [custom cron-powered schedules](#custom-cron-schedule)
+- **Continuous Integration (CI)** tab &mdash; Configure [continuous integration](/docs/deploy/cloud-ci-job) to run when someone opens a new pull request in your dbt repository
+- **API** tab &mdash; Use the [API](/docs/dbt-cloud-apis/overview) to trigger a job or send events to other systems
+
+<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/triggers.jpg" title="Configuring your job triggers"/>
+
+## Scheduling
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]: sugg clearer title
```suggestion
## Schedule jobs
```

File: website/docs/docs/deploy/job-triggers.md [hunk 7]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+
+To configure your job to run on particular day(s) and time(s), you can toggle the **Run on schedule** toggle and customize the days, time, and intervals you want your job to run.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion

To schedule your job to run at specific days, times, and intervals, go to the **Schedule** tab, and toggle **Run on schedule.** Use either the [scheduled days](#schedule-days or the [custom cron-powered schedule](#custom-cron-schedule] method to customize your desired days, times, and intervals.
```

File: website/docs/docs/deploy/job-triggers.md [hunk 8]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+
+### Schedule days
+
+Under **Timing**, you can configure your job to run either at customizable hours hours or exact intervals. Customizable hours is useful for jobs that need to run frequently throughout the day, while at exact intervals is useful for jobs that need to run at specific times of day.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
To set your job's schedule, use the **Schedule Days** option to choose specific days of the week, and select customized hours or intervals. 

Under **Timing**, you can either use customizable hours for jobs that need to run frequently throughout the day or exact intervals for jobs that need to run at specific times.
```

File: website/docs/docs/deploy/job-triggers.md [hunk 9]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+
+#### "Every N hours"
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
- **Every `n` hours** &mdash; Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs. For example, if you set it to "every 2 hours", the job will run every 2 hours from midnight UTC. This option is useful if you need to run jobs multiple times per day at regular intervals.
```

File: website/docs/docs/deploy/job-triggers.md [hunk 10]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+This option allows you to set a frequency for your job to run, based on the number of hours between each run. You can enter a number between 1 and 23, which represents the interval between each job run. For example, if you set it to "every 2 hours," the job will run every 2 hours starting from midnight UTC. This option is useful if you want to run your jobs at fixed intervals throughout the day and have them run more frequently than once a day.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion

```

File: website/docs/docs/deploy/job-triggers.md [hunk 11]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+
+#### "At exact intervals"
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
- **At exact intervals** &mdash; Use this option to set specific times when your job should run. You can enter a comma-separated list of hours (in UTC) when you want the job to run. For example, if you set it to `0,12,23,` the job will run at midnight, noon, and 11 PM UTC. This option is useful if you want your jobs to run at specific times of day and don't need them to run more frequently than once a day.
```

File: website/docs/docs/deploy/job-triggers.md [hunk 12]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+This option allows you to set specific times when your job should run. You can enter a comma-separated list of hours (in UTC) when you want the job to run. For example, if you set it to `0,12,23,` the job will run at midnight, noon, and 11 PM UTC. This option is useful if you want your jobs to run at specific times of day and don't need them to run more frequently than once a day.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion

```

File: website/docs/docs/deploy/job-triggers.md [hunk 13]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+
+:::info
+
+dbt Cloud uses [coordinated universal time](https://en.wikipedia.org/wiki/Coordinated_Universal_Time), it is not translated to your timezone or aware of daylight savings time. For examples:
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
dbt Cloud uses [Coordinated Universal Time](https://en.wikipedia.org/wiki/Coordinated_Universal_Time) (UTC) and does not account for translations to your specific timezone or take into consideration daylight savings time. For example:
```

File: website/docs/docs/deploy/job-triggers.md [hunk 14]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+
+- 0 means 12am (midnight) UTC
+- 12 means 12pm (afternoon) UTC
+- 23 means 11pm UTC
+
+:::
+
+### Custom cron schedule
+
+In dbt Cloud, you can use "cron" syntax to specify when you'd like your job to run. Cron syntax is very customizable, including the minute, hour, day of the month, month, and day of the week. This allows you to schedule jobs to run at specific times on specific days, as well as to set up more complex schedules, such as running a job on the first Monday of every month.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
To fully customize the scheduling of your job, choose the **Custom cron schedule** option and use the "cron" syntax. With this syntax, you can specify the minute, hour, day of the month, month, and day of the week, allowing you to set up complex schedules like running a job on the first Monday of each month.
```

File: website/docs/docs/deploy/job-triggers.md [hunk 15]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
+
+If you need help coming up with the right cron syntax, we recommend using a tool like [crontab.guru](https://crontab.guru). There, you can enter cron snippets and see what they mean in plain English. You can also find some example snippets below.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
Use tools such as [crontab.guru](https://crontab.guru/) to generate the correct cron syntax. This tool allows you to input cron snippets and returns their plain English translations. Refer to the following sample snippets:
```

File: website/docs/docs/deploy/job-triggers.md [hunk 16]
```diff
@@ -4,9 +4,43 @@ id: "job-triggers"
...
 
 ### Examples
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion

```

File: website/docs/docs/deploy/job-triggers.md [hunk 17]
```diff
@@ -4,11 +4,45 @@ id: "job-triggers"
...
+- **Schedule** tab &mdash; Use the **Run on schedule** toggle to configure your job to run on either simple [scheduled days](#schedule-days) or [custom cron-powered schedules](#custom-cron-schedule)
+- **Continuous Integration (CI)** tab &mdash; Configure [continuous integration](/docs/deploy/cloud-ci-job) to run when someone opens a new pull request in your dbt repository
+- **API** tab &mdash; Use the [API](/docs/dbt-cloud-apis/overview) to trigger a job or send events to other systems
+
+<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/triggers.jpg" title="Configuring your job triggers"/>
+
+## Schedule jobs
+
+
+To schedule your job to run at specific days, times, and intervals, go to the **Schedule** tab, and toggle **Run on schedule.** Use either the [scheduled days](#schedule-days or the [custom cron-powered schedule](#custom-cron-schedule] method to customize your desired days, times, and intervals.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
To schedule your job to run at specific days, times, and intervals:
1. Go to the specific job settings, click **Edit**
2. Then go to the **Triggers** section 
3. Go to the **Schedule** tab, and toggle **Run on schedule.** 
4. Use either the [scheduled days](#schedule-days or the [custom cron-powered schedule](#custom-cron-schedule] method to customize your desired days, times, and intervals.
```

**Review by ernestoongaro — DISMISSED** [review]

**Comment by mirnawong1** [comment]
hey @ernestoongaro! thank you again for the awesome insight - i think this is ready for a final docs review 🙌 . I'll go ahead and ask for one but wanted to let you know in case you wanted a final look at the [deploy preview](https://deploy-preview-2944--docs-getdbt-com.netlify.app/docs/deploy/job-triggers#schedule-days)

**Comment by ernestoongaro** [comment]
@mirnawong1 one final review and I added a quick change to point to the Admin API (as that is what is relevant for scheduling). I also want to point out there is a file edited here that doesn't seem to be related `website/sidebars.js`

Besides that I'm good to go!

**Comment by mirnawong1** [comment]
> @mirnawong1 one final review and I added a quick change to point to the Admin API (as that is what is relevant for scheduling). I also want to point out there is a file edited here that doesn't seem to be related `website/sidebars.js`
> 
> Besides that I'm good to go!

great thank you @ernestoongaro . the sidebar.js change was from yesterday when i was trying to fix the failed checks 👍

File: website/docs/docs/deploy/job-triggers.md [hunk 18]
```diff
@@ -1,14 +1,56 @@
...
 
-If you need help coming up with the right cron syntax, we recommend using a tool like `crontab.guru`. There, you can enter cron snippets and see what they mean in plain English. You can also find some example snippets below.
+- **Schedule** tab &mdash; Use the **Run on schedule** toggle to configure your job to run on either [scheduled days](#schedule-days) or [custom cron-powered schedule](#custom-cron-schedule)
+- **Continuous Integration (CI)** tab &mdash; Configure [continuous integration](/docs/deploy/cloud-ci-job) to run when someone opens a new pull request in your dbt repository
+- **API** tab &mdash; Use the [API](/docs/dbt-cloud-apis/admin-cloud-api) to trigger a job or send events to other systems
+
+<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/triggers.jpg" width="85%" title="Configuring your job triggers"/>
+
+## Schedule jobs
+
```
Comment (runleonarun) [inline website/docs/docs/deploy/job-triggers.md:16]:
```suggestion
```

File: website/docs/docs/deploy/job-triggers.md [hunk 19]
```diff
@@ -1,14 +1,56 @@
 ---
 title: "Job triggers"
 id: "job-triggers"
-description: "You can use cron syntax to specify when you want to run a job."
+description: "You can configure when and how dbt should run your job"
 ---
 
-In dbt Cloud, you can use "cron" syntax to specify when you'd like your job to run. Cron syntax is very expressive, and allows you to completely customize your run schedule.
+In dbt Cloud, you can use the options under **Triggers** to configure when and how dbt should run your job:
```
Comment (runleonarun) [inline website/docs/docs/deploy/job-triggers.md]:
```suggestion
In dbt Cloud, you can use the options under **Triggers** to configure when and how dbt should [run your job](/docs/deploy/job-triggers#schedule-job):
```

**Review by runleonarun — APPROVED** [review]
I just made [this tiny change](https://github.com/dbt-labs/docs.getdbt.com/pull/2944/commits/acc01a24c8b25c955992686d7d3a613e30ae402b), but looks great! :shipit:
